assets/airlines/swn.json

```diff
@@ -0,0 +1,17 @@
+{
+    "icao": "swn",
+    "name": "West Air Sweden",
+    "callsign": {
+        "name": "Air Sweden",
+        "callsignFormats": [
+            "####"
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["B762", 4],
```

Comment:
In more detail they use 767-200SF, is it okay to just use a 762 as an approximation?